nibabel/spatialimages.py

```diff
@@ -656,7 +656,7 @@ def from_image(klass, img):
                      klass.header_class.from_header(img.header),
                      extra=img.extra.copy())
 
-    def __getitem__(self):
+    def __getitem__(self, idx=None):
```

Comment:
Oops - well spotted.  But - is `__getitem__` ever called without arguments?  I mean, why not just:

```
def __getitem__(self, idx):
```

?